common/app/layout/Sublink.scala

```diff
@@ -61,7 +62,7 @@ case class Sublink(
   cardStyle: CardStyle,
   mediaType: Option[MediaType],
   pillarName: String,
-  contentType: String
+  designType: String
```

Comment:
I would store the DesignType rather than a string here. Compiler will warn you about some potential typo when comparing it

Reply:
Ah OK, I did this so I could move the `getOrElse` check further upstream into this class rather than in a few places that use `designType`.

Reply:
@TBonnin is there another way that you'd recommend which would allow me define the fallback string used `pillar`/`designType` once, rather than writing `item.pillar.map(_.name).getOrElse("News").toLowerCase()` and `item.designType.map(_.toString).getOrElse("Article").toLowerCase()` wherever these properties are used? Or is that the best approach to take?

Reply:
we can probably enrich those type to add a `nameOrDefault` funtion that would contain this logic. Let me know when you are available we can pair to do this.